### PR TITLE
Add `compute` method for dask.imperative

### PR DESCRIPTION
**Pull request by jcrist**
Can now compute multiple values at once in a clean manner.

```
>>> from dask.imperative import value, compute
>>> a = value(1)
>>> b = a + 1
>>> c = a + 2
>>> compute(b, c)
(2, 3)
```

cc @cpcloud

### CODE REVIEW

File: dask/imperative.py [hunk 1]
```diff
@@ -183,6 +183,25 @@ def get(dsk, keys, get=None, **kwargs):
...
+    Examples
+    --------
+    >>> a = value(1)
+    >>> b = a + 2
+    >>> c = a + 3
+    >>> compute(b, c)  # Compute both simultaneously
+    (3, 4)
+    """
+    if len(args) == 1:
+        return args[0].compute(**kwargs)
```
Comment (mrocklin) [inline dask/imperative.py]: Recommend either always returning a tuple or changing the other `compute` functions to have this policy.

**Comment by mrocklin** [comment]
+1 other than the remark in compute.

File: dask/imperative.py [hunk 2]
```diff
@@ -183,6 +183,25 @@ def get(dsk, keys, get=None, **kwargs):
...
+
+    Examples
+    --------
+    >>> a = value(1)
+    >>> b = a + 2
+    >>> c = a + 3
+    >>> compute(b, c)  # Compute both simultaneously
+    (3, 4)
+    """
+    if len(args) == 1:
```
Comment (shoyer) [inline dask/imperative.py]: I don't like this sort of inference -- better to always return a tuple. The `compute` method can always be used if you want a single value, and this makes it hard to use programmatically.

File: dask/imperative.py (hunk 1 above)
Comment (shoyer) [inline dask/imperative.py]: I would not change the other compute functions. I argued against this pretty hard for `dask.array` as well :).

Comment (jcrist) [inline dask/imperative.py]: I do like having type-stable functions. However, I did this specifically for the following:

```
>>> a = value(1)
>>> b = a + 1
>>> c = a + 2
>>> rb, rc = compute(b, c)    # Works, is integer
>>> rb = compute(b)    # Works, is integer
```

However, I don't feel too strongly about this. You're correct that `.compute` can always be used for the single element case.

Comment (shoyer) [inline dask/imperative.py]: The problem really comes up when you are passing in `*args` to compute, e.g.,
```
>>> results = compute(*args)
```

Now I have to write something special to ensure that `results` is a tuple.

**Pull request merged**